src/app/pages/search-character/search-character.component.html

```diff
@@ -9,8 +9,9 @@ <h1 class="title">Encuentra a tu personaje favorito</h1>
       (onGenderSelectorChange)="handleGenderChange($event)"
       (onSpecieSelectorChange)="handleSpecieChange($event)"
     ></app-big-input>
-    <ng-container *ngIf="charactersList.length > 0">
+    <div class="results" *ngIf="charactersList.length > 0">
```

Comment:
- This is where I mention the conditional that would not be necessary for the counter button